packages/dependency-royalty-management/contracts/RoyaltySplitter.sol

```diff
@@ -183,7 +183,7 @@ contract RoyaltySplitter is Initializable, OwnableUpgradeable, IRoyaltySplitter,
         );
         require(
             !callData.startsWith(IERC20Approve.approve.selector) &&
-                !callData.startsWith(IERC20Approve.increaseAllowance.selector),
+                !callData.startsWith(IERC20Approve.transfer.selector),
```

Comment:
@rishabh0x00 how come this change doesn't have any impact on the Asset tests?

Reply:
What prompted this change?

Reply:
Though I would change this back I had a Hunch but this change is not required.